libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -132,6 +139,7 @@ dictionary Config {
     u32 payment_timeout_sec;
     string? default_lsp_id;
     string? api_key;
+    boolean restore_only;
```

Comment:
It seems as the `restore_only` flag is better fit as a parameter to the connect function rather than adding it to the config. Nevertheless I see your concern about adding another parameter to the connect.

Reply:
Yes, I wanted to avoid adding a restore_only param or options struct, which seems excessive.

The other option would be a ConnectRequest:
```
pub struct ConnectRequest {
    pub config: Config,
    pub seed: Vec<u8>,
    pub restore_only: Option<bool>,
}

pub async fn connect(
    req: ConnectRequest,
    event_listener: Box<dyn EventListener>,
) -> Result<Arc<BreezServices>, ConnectError>
```

Reply:
I am in favor of ConnectRequest. Also more consistent with other methods.
Why not inserting the event listener inside the ConnectRequest like other parameters?

Reply:
I can test, but I think it might break the RN code generation

Reply:
In that case I think we can go with the listener outside the struct.